customer/src/main/java/com/yas/customer/controller/CustomerController.java

```diff
@@ -54,6 +54,14 @@ public ResponseEntity<CustomerVm> getCustomerProfile() {
         return ResponseEntity.ok(customerService.getCustomerProfile(SecurityContextHolder.getContext().getAuthentication().getName()));
     }
 
+    @GetMapping("/storefront/customer/{customerId}")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "Ok", content = @Content(schema = @Schema(implementation = CustomerVm.class))),
+           })
+    public ResponseEntity<CustomerVm> getCustomerById(@PathVariable String customerId) {
```

Comment:
You should not create new API (start with /storefront) to get customer data.
Please refer to com.yas.customer.config.SecurityConfig.java, all requests start with /storefront are not authorized
![image](https://user-images.githubusercontent.com/5679842/222132110-7bb87c02-dfcf-45d6-b9d9-6c9d4bd24bd9.png)

You should create API with path /customers/{customerId} then provide access_token when call it. (Refer com.yas.product.service.MediaService.removeMedia)

![image](https://user-images.githubusercontent.com/5679842/222133087-c34cb697-95c7-4404-88f8-54e6e56dcb48.png)